supernode/server/router.go

```diff
@@ -20,6 +21,7 @@ func initRoute(s *Server) *mux.Router {
 	handlers := []*HandlerSpec{
 		// system
 		{Method: http.MethodGet, Path: "/_ping", HandlerFunc: s.ping},
+		{Method: http.MethodGet, Path: "/version", HandlerFunc: version.HandlerWithCtx},
```

Comment:
If you add this HTTP endpoint, you need to add this route into https://github.com/dragonflyoss/Dragonfly/blob/master/apis/swagger.yml first.

Reply:
I have already added the definition to swagger.yml. Do I have to generate the docs?

Reply:
Yes, and you can do that by exec the shell `./hack/generate-swagger-models.sh`.

Reply:
Thanks for your reply